Makefile

```diff
@@ -132,7 +132,7 @@ uninstall: manifests $(KUSTOMIZE) ## Uninstall CRDs from a cluster
 
 .PHONY: deploy
 deploy: manifests $(KUSTOMIZE) ## Deploy controller in the configured Kubernetes cluster in ~/.kube/config
-	cd config/manager && $(ROOT)/$(KUSTOMIZE) edit set image controller=${IMG}
+	cd config/manager && $(ROOT)/$(KUSTOMIZE) edit set image controller=$(subst :v,:,$(IMG))
```

Comment:
Nit, but we could do `$(subst operator:v,operator:` to highlight clearly what this line does, or comment above